package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
```

Comment:
Can we fix the package xml to look more like:
https://github.com/graspit-simulator/graspit-ros/blob/master/graspit/package.xml